### DISCUSSION TITLE
Проблема с установкой zapret на Keenetic

### DISCUSSION THREAD

**Discussion by s9900s** [bol-van/zapret#863] [Q&A]
Делаю всё по  инструкции, но на этапе запуска выдаёт следующее:
/opt # mv zapret-master zapret
/opt # cd zapret
/opt/zapret # ./install_bin.sh
no compatible binaries found

Файлы все на месте:
/opt/zapret # ls
Makefile           files              ip2net             tpws
blockcheck.sh      init.d             ipset              uninstall_easy.sh
common             install_bin.sh     mdig
config.default     install_easy.sh    nfq
docs               install_prereq.sh  tmp
/opt/zapret #

Что я не так делаю?

**Comment by bol-van** [comment]
Инструкции в сети устарели. Бинарики убраны из репы. Нужно качать релиз

**Comment by s9900s** [comment]
> Инструкции в сети устарели. Бинарики убраны из репы. Нужно качать релиз

Можете подсказать, как это правильно сделать?

**Comment by IndeecFOX** [comment]
@s9900s 
https://habr.com/ru/articles/834826/
Примерно так
